plugins/accuracy.cc

```diff
@@ -243,15 +248,15 @@ runBoolean(AccuracyConfig & runAccuracyConf,
 RunOutput
 runCategorical(AccuracyConfig & runAccuracyConf,
                BoundSelectQuery & selectQuery,
-               std::shared_ptr<Dataset> output)
+               std::shared_ptr<Dataset> output,
+               int accuracyOverN)
 {
-    typedef vector<std::tuple<CellValue, CellValue, double, double, RowPath>> AccumBucket;
+    typedef vector<std::tuple<CellValue, CellValue, std::vector<CellValue>, double>> AccumBucket;
```

Comment:
This will slow everything down, massively.  We are copying a vector on every accumulation.  Instead, make it take a `const CellValue *` or ideally a `double *` and a size_t, and have a separate path for each possibility.